spiceaidocs/docs/features/cdc/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: 'Change Data Capture (CDC)'
+sidebar_label: 'Change Data Capture'
+description: 'Learn how to use Change Data Capture (CDC) in Spice.'
+sidebar_position: 6
+pagination_prev: null
+pagination_next: null
+---
+
+Change Data Capture (CDC) is a technique to capture changed rows from a database's transaction log and deliver to consumers with low latency. Leveraging this technique allows Spice to keep [locally accelerated](../local-acceleration/index.md) datasets up-to-date in real-time with the source data, and is highly efficient by only transferring the changed rows instead of re-fetching the entire dataset on refresh.
```

Comment:
>  Leveraging this technique allows Spice to keep [locally accelerated](../local-acceleration/index.md) datasets up-to-date in real-time with the source data, and is highly efficient by only transferring the changed rows instead of re-fetching the entire dataset on refresh.

Redundancy between this and ##Benefits

Reply:
Repetition is the key to mastery! (I did this intentionally)